src/components/Modal/Modal.stories.tsx

```diff
@@ -1,83 +1,133 @@
-import { useArgs } from '@storybook/client-api';
+import Button from '@components/Button';
+import Container from '@components-layout/Container';
+import Flexbox from '@components-layout/Flexbox';
+import { css } from '@emotion/react';
 import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { MouseEventHandler, useState } from 'react';
 
-import Modal, { Button } from '.';
+import Modal from '.';
 
 export default {
-  title: 'Modal',
+  title: 'Design System/Components/Modal',
```

Comment:
ㅠㅠ 제가 늦게 Template 올리는 바람에..
사이드바 depth 를 줄이기로 했습니다!
`Design System` 을 삭제헤주세요~

```suggestion
  title: 'Components/Modal',
```